outwatch/src/main/scala/outwatch/dom/helpers/Builder.scala

```diff
@@ -24,7 +24,14 @@ object AttributeBuilder {
 
 trait AccumulateAttrOps[T] { self: AttributeBuilder[T, BasicAttr] =>
   protected def name: String
-  @inline def accum(s: String): AccumAttrBuilder[T] = accum(_ + s + _)
+  @inline def accum(s: String): AccumAttrBuilder[T] = new AccumAttrBuilder[T](
+    name, 
+    this, 
+    {
+      case (v1, v2) => s"$v1$s$v2"
+    }
+  )
+
```

Comment:
Why was this code changed? I kind of liked the older version better :)

Reply:
see 217803

Reply:
I suppose it could have been `_.toString + _.toString`?

Reply:
Oh, okay. I guess that is even a good thing. So nevermind!